provisioning/device/src/polling_state_machine.ts

```diff
@@ -128,10 +132,9 @@ export class PollingStateMachine extends EventEmitter {
         responseComplete: {
           _onEnter: (result, response, callback) => {
             this._currentOperationCallback = null;
-            /* Codes_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_014: [ If `PollingTransport.registrationRequest` succeeds with status==Assigned, it shall emit an 'operationStatus' event and call `callback` with null, the response body, and the protocol-specific result. ] */
-            /* Codes_SRS_NODE_PROVISIONING_TRANSPORT_STATE_MACHINE_18_020: [ If `PollingTransport.queryOperationStatus` succeeds with status==Assigned, `register` shall emit an 'operationStatus' event and complete and pass the body of the response and the protocol-specific result to the `callback`. ] */
-            this.emit('operationStatus', result);
-            this._fsm.transition('idle', null, result, response, callback);
```

Comment:
seems like a behavior change to remove this. is this intentional? why?

Reply:
1) It was intentional.
2) Nobody uses this.
3) You actually told me to do this the last time.

---
In reply to: [282191372](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282191372) [](ancestors = 282191372)

Reply:
ok i guess.

---
In reply to: [282192386](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282192386) [](ancestors = 282192386,282191372)